style.css

```diff
@@ -0,0 +1,100 @@
+body {
+    font-family: 'Comfortaa', cursive;
+}
+
+h2 {
+  font-size: 40px;
+}
+
+.blue-box {
+   background-color: lightblue;
+   padding: 110px;
+   border-radius: 30px;
+   text-align: center;
+   border: 15px solid pink;
+   width: 20%;
+   height: 40%;
+ }
+
+ ul {
+   list-style-type: none;
+   margin: 0;
+   padding: 0;
+   overflow: hidden;
+   text-decoration: underline;
+   color: black;
+ }
+
+ li {
+   display: inline;
+   float: left;
+ }
+
+ a {
+   display: block;
+   padding: 10px;
+   color: black;
+   text-underline-position: under;
+ }
+
+ a:hover {
+   color: purple;
+ }
+
+ .pink-box {
+   background-color: pink;
+   color: black;
+   padding: inherit;
+   text-align: center;
+   border-radius: 10px;
+   width: 200px;
+   height: 40px;
+   overflow: hidden;
+   margin: auto;
+}
+
+button {
+  background-color: darkblue;
+  color: white;
+  border: 1px black solid;
+  box-shadow: 5px 10px 8px #888888;
+  margin: auto;
+  padding: 8px 12px;
+  font-size: 20px;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.guess:hover {
+   background:lighten(darkblue, 20%);
+   margin:4px;
+   padding:14px 28px;
+   webkit-transition: all 200ms ease;
+}
+
+input {
+  width: 35px;
+  height: 55px;
+}
+
+h1 {
+  font-size: 60px;
+  font-stretch: extra-expanded;
+  font-style: normal;
+  font-weight: 700px;
+  overflow: hidden;
+  float: left;
+  display: inline-block;
+}
+
+.image {
+  width: 400px;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  justify-content: center;
+}
+
+h3 h4 {
```

Comment:
Oh, I think you need a comma here to separate h3 and h4. The code currently means that you're applying style to all `h4`s that are inside an `h3`.
More info here https://www.w3schools.com/cssref/css_selectors.asp